### ISSUE TITLE
Re-organize the items in Downloads and Documents section of frame-next.eu

### ISSUE THREAD

**Issue by mertaksac** [FRAME-NEXT/FRAME#75]
Below comment is from Richard:

`The web page needs to be re-organised so that the guides and the tool files are at the top and you have to scroll down to find everything else.  Also the web page does not contain any instructions for what to do when you have downloaded the files, or even a “read me first” indication, which I think would be helpful for the un-initiated.`

**Comment by mertaksac** [comment]
Please kindly comment if this could be marked as closed

**Comment by burespe1** [comment]
FRAME-NEXT was a project, FRAME is a product. we need to maintain FRAME website and archive or make onepager of the now obsolete and hopelesly unfinished FRAME-NEXT website. I ceated a new issue to this so we can get rid of this duplicity #94.

**Comment by bossomr** [comment]
I agree with the idea that we need some instructions to guide users in what to do once they have downloaded the files from the FRAME web site.  This is something that I think should be done as part of the web site upgrade implied by Petr's comment above.  If the instructions are in a "read me" document that regular users can skip it, so long as this document does not contain any descriptions of the changes that have been made to create the current version.  These should be in a separate document, which could be named "release notes", or something similar.
